app/library/template.hbs

```diff
@@ -0,0 +1,6 @@
+
+<div class="zybooks">
+{{#each model as |libraryUnit|}}
```

Comment:
Instead of `|libraryUnit|` consider using a more descriptive name like `|zybook|`